project/settings.py

```diff
@@ -125,8 +130,8 @@ def get(variable):
         'default': {
             'ENGINE': 'django.db.backends.postgresql',
             'NAME': 'chatter',
-            'USER': 'django',
-            'PASSWORD': 'P@ssword1'
+            'USER': 'postgres',
```

Comment:
haha, didn't have to do this, but it is appreciated 🙇

Reply:
Not a problem! Good postgres practice 👌